src/main/java/net/fabricmc/loader/impl/FabricLoaderImpl.java

```diff
@@ -77,10 +77,9 @@ public final class FabricLoaderImpl extends net.fabricmc.loader.FabricLoader {
 	public static final String VERSION = "0.15.7";
 	public static final String MOD_ID = "fabricloader";
 
-	public static final String CACHE_DIR_NAME = ".fabric"; // relative to game dir
-	private static final String PROCESSED_MODS_DIR_NAME = "processedMods"; // relative to cache dir
-	public static final String REMAPPED_JARS_DIR_NAME = "remappedJars"; // relative to cache dir
-	private static final String TMP_DIR_NAME = "tmp"; // relative to cache dir
+	private static final String PROCESSED_MODS_DIR_NAME = "processedMods"; // relative to loader cache dir
+	public static final String REMAPPED_JARS_DIR_NAME = "remappedJars"; // relative to loader cache dir
+	private static final String TMP_DIR_NAME = "tmp"; // relative to loader cache dir
```

Comment:
Again, see above comment, a tmp dir in .cache makes little sense. Lets not move these.